maxim/logger/livekit/stt.py

```
@@ -137,7 +135,7 @@ def pre_hook(self, hook_name, args, kwargs):
         )
 
 
-def post_hook(self, result, hook_name, args, kwargs):
+def post_hook(self: STT, result, hook_name, args, kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `post_hook`

Add return type annotation: `None`

(ANN201)